src/pymor/__init__.py

```diff
@@ -3,6 +3,18 @@
 # License: BSD 2-Clause License (http://opensource.org/licenses/BSD-2-Clause)
 
 import os
+import platform
+import sys
+
+
+if sys.version_info.major != 3:
```

Comment:
Not sure if it's still current, but the version after 3.9 once was supposed to be 4.0. Maybe make these checks more forward compatible?

Reply:
Changed.